src/app/printer-status/printer-status.component.html

```diff
@@ -1,25 +1,74 @@
 <table class="printer-status" *ngIf="printerStatus">
     <tr>
         <td class="printer-status__value printer-status__value-1"
-            [ngClass]="{'printer-status__no-set-value': printerStatus.nozzle.set === 0}">
+            [ngClass]="{'printer-status__no-set-value': printerStatus.nozzle.set === 0}"
+            (click)="showQuickControlHotend()" matRipple [matRippleUnbounded]="false">
             <img src="assets/nozzle.svg">
             <span class="printer-status__actual-value">{{ printerStatus.nozzle.current }}<span
                     class="printer-status__unit">°C</span></span>
             <span class="printer-status__set-value"
                 *ngIf="printerStatus.nozzle.set">/{{ printerStatus.nozzle.set }}<span class="unit">°C</span></span>
         </td>
         <td class="printer-status__value printer-status__value-2"
-            [ngClass]="{'printer-status__no-set-value': printerStatus.heatbed.set === 0}">
+            [ngClass]="{'printer-status__no-set-value': printerStatus.heatbed.set === 0}"
+            (click)="showQuickControlHeatbed()" matRipple [matRippleUnbounded]="false">
             <img src="assets/heat-bed.svg">
             <span class="printer-status__actual-value">{{ printerStatus.heatbed.current }}<span
                     class="printer-status__unit">°C</span></span>
             <span class="printer-status__set-value"
                 *ngIf="printerStatus.heatbed.set">/{{ printerStatus.heatbed.set }}<span class="unit">°C</span></span>
         </td>
-        <td class="printer-status__value printer-status__value-3 printer-status__no-set-value">
+        <td class="printer-status__value printer-status__value-3 printer-status__no-set-value"
+            (click)="showQuickControlFan()" matRipple [matRippleUnbounded]="false">
             <img src="assets/fan.svg">
             <span class="printer-status__actual-value">{{ printerStatus.fan }}<span
                     class="printer-status__unit">%</span></span>
         </td>
     </tr>
-</table>
\ No newline at end of file
+</table>
+<div class="quick-control" *ngIf="view !== QuickControlView.NONE" id="quickControl">
+    <table class="top-bar" (click)="hideQuickControl()">
+        <tr>
+            <td class="top-bar__back"><img src="assets/back.svg" class="top-bar__back-icon" matRipple
+                    [matRippleUnbounded]="false" />back</td>
+            <img [src]="'assets/' + (view === QuickControlView.HOTEND ? 'nozzle.svg' : view === QuickControlView.HEATBED ? 'heat-bed.svg' : 'fan.svg')"
+                class="quick-control__center-icon-small">
+        </tr>
+    </table>
+    <div class="quick-control__controller">
```

Comment:
```suggestion
            <td class="top-bar__back"><img src="assets/back.svg" class="top-bar__back-icon" matRipple
                    [matRippleUnbounded]="false" />back</td>
        </tr>
    </table>
    <img [src]="'assets/' + (view === QuickControlView.HOTEND ? 'nozzle.svg' : view === QuickControlView.HEATBED ? 'heat-bed.svg' : 'fan.svg')"
        class="quick-control__center-icon-small">
    <div class="quick-control__controller">
```

Currently the icon doesn't display. Moving rows 34/35 down 2 rows so the image is outside the table fixes the issue.

Alternatively, changing the class to top-bar__center-icon and keeping it where it is inside the table row gets it to display although it then isn't centered.